common/src/test/java/com/microsoft/identity/common/nativeauth/util/ApiConstants.kt

```diff
@@ -29,6 +29,7 @@ import java.net.URL
  */
 interface ApiConstants {
     companion object {
+        // TODO: Update mock api url
```

Comment:
And this

Reply:
If we can't remove/resolve this TODO now, please add a link to the respective ADO PBI to the code, for context.

Reply:
Removed the TODO and updated the url